tfjs-backend-webgpu/src/kernels/ToPixels.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use backend file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {KernelConfig, KernelFunc, TensorInfo} from '@tensorflow/tfjs-core';
+import {ToPixels, ToPixelsAttrs, ToPixelsInputs} from '@tensorflow/tfjs-core';
+
+import {TextureInfo, WebGPUBackend} from '../backend_webgpu';
+import {ToPixelsProgram} from '../to_pixels_webgpu';
+
+export function toPixels(
+    args:
+        {inputs: ToPixelsInputs, backend: WebGPUBackend, attrs: ToPixelsAttrs}):
+    TensorInfo {
+  const {inputs, backend, attrs} = args;
+  const {$img} = inputs;
+  const {canvas, numChannels} = attrs;
+  const [height, width] = $img.shape.slice(0, 2);
+
+  const format = 'rgba8unorm';
+  const outShape = [height, width, 1];
+  const program = new ToPixelsProgram(outShape, $img.dtype, format);
+  canvas.width = width;
+  canvas.height = height;
+  const gpuContext = canvas.getContext('webgpu');
```

Comment:
What if  another context (`2d` or `webgl`) has been requested for `canvas` and `canvas.getContext('webgpu')` return null?